pkg/northbound/admin/admin.proto

```diff
@@ -16,8 +16,8 @@ limitations under the License.
 
 syntax = "proto3";
 
-// Package admin defines the administrative and diagnostic gRPC interfaces.
-package proto;
+// Package admin defines the administrative gRPC interfaces.
+package onos.config.admin;
```

Comment:
is this what we want ? also please do update the comment

Reply:
I was copying the style from Jordan's types.proto